runtime/gc_realtime/RealtimeAccessBarrier.hpp

```diff
@@ -152,6 +154,9 @@ class MM_RealtimeAccessBarrier : public MM_ObjectAccessBarrier
 	virtual I_32 backwardReferenceArrayCopyIndex(J9VMThread *vmThread, J9IndexableObject *srcObject, J9IndexableObject *destObject, I_32 srcIndex, I_32 destIndex, I_32 lengthInSlots);
 	virtual I_32 forwardReferenceArrayCopyIndex(J9VMThread *vmThread, J9IndexableObject *srcObject, J9IndexableObject *destObject, I_32 srcIndex, I_32 destIndex, I_32 lengthInSlots);
 
+	virtual BOOLEAN checkStringConstantsLive(J9JavaVM *javaVM, j9object_t stringOne, j9object_t stringTwo);
+	virtual BOOLEAN checkStringConstantLive(J9JavaVM *javaVM, j9object_t string);
```

Comment:
I'd actually use _bool_ in C++ part, and BOOLEAN only in C wrapper. Conversion from bool to BOOLEAN should not be a problem.